.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Potential Bug", "To Triage"]
+assignees: achakko
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please contact the Matomo community forum for questions: https://forum.matomo.org/
+        If this bug is regarding a premium plugin you purchased on plugins.matomo.org, please email shop@matomo.org instead of creating a bug report here.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please tell us what happens instead of the expected behavior.
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: what-should-actually-happen
+    attributes:
+      label: What should happen?
+      description: Please tell us what should happen. How has this issue affected you? What are you trying to accomplish?
+      placeholder: Tell us what you expected to see!
+      value: "This is what should happen!"
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: How can this be reproduced?
+      description: For UI bugs, please try to reproduce that on our demo (https://demo.matomo.cloud/) If that is possible, please include the URLs and steps how to reproduce it there. For other bugs or UI bugs that cant be reproduced on Demo, please include any relevant URLs or commands your are using that may help us to reproduce the bug
+      placeholder: Steps to Reproduce
+      value: "This is how it happens!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Matomo Version
+      description: What version of Matomo are you running?
+      options:
+        - Matomo 3
+        - Matomo 4
```

We should include Matomo 5 RC for now and later remove the RC once stable version is released.
```suggestion
        - Matomo 3
        - Matomo 4
        - Matomo 5 RC
```

---

We shouldn't make that a drop down at all. We wan't to know the exact version.

---

Yeah, perhaps we can change this to a required text input, so that people make the effort to try and find the version. They will write n/a or don't know or a dot etc. when they don't know it. But if we make it optional people won't provide at times it even when they know it.

---

Will leave two required fields for major version and minor version.